pkg/debuginfo/extract.go

```diff
@@ -41,25 +39,6 @@ func NewExtractor(logger log.Logger) *Extractor {
 	}
 }
 
-// ExtractAll extracts debug information from the given executables.
-// It consumes a map of file sources to extract and a destination io.Writer.
-func (e *Extractor) ExtractAll(ctx context.Context, srcDsts map[string]io.WriteSeeker) error {
```

Comment:
@javierhonduco This is used in https://github.com/parca-dev/parca-debuginfo/blob/15a2e2233ea8124ca3a38fde4ffdd75bf407f06d/cmd/parca-debuginfo/main.go#L171

Reply:
BTW We can consider to move it to the call site